### ISSUE TITLE
[Suggestion] Improve accessibility for reading 

### ISSUE THREAD

**Issue by gn-bcampbell** [YourAverageTechBro/DevToolboxWeb#79]
### Suggested change
- Improve the readability of text within the application.
- Make a distinction between the tooling and the GitHub repo redirect.

### Why
- The black font on a dark gray background is quite difficult to read and raises accessibility concerns.

### Comments
- This happens elsewhere in the app but for now this PR makes minor changes to the tool list on the main page.
- If this suggestion is helpful, I'll be happy to look into other sections of the app - maybe even introduce a theme picker🌈 

#### Before
![image](https://github.com/YourAverageTechBro/DevToolboxWeb/assets/80278654/a23feb95-f657-474b-beb9-d860d74b9de8)
#### After
![image](https://github.com/YourAverageTechBro/DevToolboxWeb/assets/80278654/856b2e8d-c2ad-44e7-8751-38d7df220641)

**Comment by gn-bcampbell** [comment]
Opened PR: https://github.com/YourAverageTechBro/DevToolboxWeb/pull/78
